tests/workflows/test_clearlabs.yml

```diff
@@ -486,7 +486,7 @@
     - path: miniwdl_run//wdl/tasks/task_sc2_gene_coverage.wdl
       md5sum: e8310700d96d98cd16f4859f61dd179f
     - path: miniwdl_run/wdl/tasks/task_taxonID.wdl
-      md5sum: ac535cb1d1a51a5ca578cb26714a49a7
+      md5sum: 7c7578f557447b23d3a91cb19a066c10
```

Comment:
Hah! Was so confused at first as to why these changes would impact the tests, but I get it now: Freyja is within taxonID and these tests are checking that files hash. Hah, looking forward to PHB as these will be much more intuitive!